lib/services/db.dart

```diff
@@ -0,0 +1,36 @@
+import 'package:cloud_firestore/cloud_firestore.dart';
```

Comment:
Let's avoid abbreviations and change this to `database.dart` - it's not too long

Reply:
Yes definitely. I was working on an old branch and there was another database.dart so I used db.dart for now.